src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -716,6 +717,19 @@ std::vector<MessagePtr> IrcMessageHandler::parseNoticeMessage(
     {
         return {generateBannedMessage(true)};
     }
+    else if (message->content().startsWith("You are timed out for "))
```

Comment:
There's a tag inside tags() with the key `msg-id` and value `msg_timedout` that we can use instead of doing a string comparison - other than that :+1: